packages/u/umockdev/package.yml

```diff
@@ -13,6 +14,7 @@ optimize   :
 builddeps  :
     - pkgconfig(gio-unix-2.0)
     - pkgconfig(gtk-doc)
+    - pkgconfig(libeconf)
```

Comment:
`pkgconfig(libeconf)` is provided by `libeconf-devel` which is dependency of `util-linux-devel`.
`util-linux-devel` is part of `system.devel` componont. So, there is no need to explicitly add `pkgconfig(libeconf)`  as build dependency.